README.md

```diff
@@ -12,31 +12,31 @@ Add a step like this to your workflow:
 ```yaml
 - uses: EndBug/add-and-commit@v4 # You can change this to use a specific version
   with:
-    # The arguments for the git add command (see the paragraph below for more info)
+    # The arguments for the `git add` command (see the paragraph below for more info)
     # Default: '.'
     add: 'src'
 
     # The name of the user that will be displayed as the author of the commit
     # Default: author of the commit that triggered the run
     author_name: Your Name
 
-    # The The email of the user that will be displayed as the author of the commit
+    # The email of the user that will be displayed as the author of the commit
     # Default: author of the commit that triggered the run
     author_email: mail@example.com
 
     # The local path to the directory where your repository is located. You should use actions/checkout first to set it up
     # Default: '.'
     cwd: './path/to/the/repo'
 
-    # Whether to use the --force option on git add, in order to bypass eventual gitignores
+    # Whether to use the --force option on `git add`, in order to bypass eventual gitignores
     # Default: false
     force: true
 
     # The message for the commit
     # Default: 'Commit from GitHub Actions'
     message: 'Your commit message'
 
-    #  The arguments for the git rm command (see the paragraph below for more info)
+    #  The arguments for the `git rm` command (see the paragraph below for more info)
```

Comment:
Why did you put the backticks here? If this was plain text that would correctly format it as code, but here it's not...

Reply:
Grammarly was confused by the sentence:

> The arguments for the git add command (see the paragraph below for more info)

It parsed add as binding to command. My initial fix was changing it to git-add which made Grammarly happy, but I worried someone would complain that git-add isn't a command.

Reply:
Hm, I see. The only alternative would be using normal quotes but that wouldn't be much better. I think it's better to keep the backticks then.